recipes/noteslib/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "noteslib" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "ca2a4de4a6982785a0160f96085fbc83630fdf0a1f8ed5a9c88a4dd5c3e93ca6"
```

Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: ca2a4de4a6982785a0160f96085fbc83630fdf0a1f8ed5a9c88a4dd5c3e93ca6
```